README.md

```diff
@@ -150,6 +129,79 @@ If CallKit has been set up correctly, Tom's device will ring like a phone call,
 
 Note: Due to an issue with ARKit, you cannot start a Remote Streem from a view using the camera https://forums.developer.apple.com/message/411888#411888
 
+#### Roles
+
+Roles dictate which side of the streem your on as in whether you're sharing your video or receiving video. They also affect which tools are available to you. The different roles are:
+
+* `LOCAL_CUSTOMER`: This is the primary role for the customer in a two-way streem
+* `ONSITE_CUSTOMER`: This is the primary role for a customer in an onsite (one-way) streem 
+* `REMOTE_PRO`: This is the primary role for the pro in a two-way streem
+* `ONSITE_PRO`: This is the primary role for a pro in an onsite (one-way) streem
+
+In general if you are starting a remote streem you will want the `LOCAL_CUSTOMER` role. 
+
+### Getting Remote User IDs
+
+In order to start a remote streem, your app will need to supply the `remoteUserId` of the user your app's user is going to streem with. There are two mechanisms available in the SDK for retrieving a `remoteUserId`. The first is through getting a list of recently logged in users. The second is through an invitation. In addition to these two methods, you can maintain your own list of `remoteUserIds` and supply them to your app however you choose. 
+
+#### Recently Logged In Users
+
+By calling the SDK's `getRecentlyIdentifiedUsers` method on the Streem `sharedInstance` you will receive a list of recently logged in users for your company. You can use this method to filter only experts. You then give this method a callback where you select the remote user through some mechanism and start the streem. 
+
+```swift
+    Streem.sharedInstance.getRecentlyIdentifiedUsers(onlyExperts: true) { [weak self] users in
+        guard let self = self else { return }
+
+        // Select the user you wish to streem with
+
+        Streem.sharedInstance.startRemoteStreem(
+            asRole: .LOCAL_CUSTOMER,
+            withRemoteUserId: selectedUser.id) { success in 
+                // handle succes of streem
+        }
+``` 
+
+#### Invitations
+
+If you are utilizing Streem's pro implementations on the web and/or iOS you will likely have access to invitations. Invitations are communicated via a 9-digit code. This code can be transmitted through SMS, email, or copy and pasted to some other mechanism such as Slack. 
+
+Once you've retrieved this 9-digit code in your app you will follow three steps. First you will call `login(withSmsInvitationId:)`. This will retrieve the invitation details for the given code. Then you will need to call `identifyCustomer(companyCode:inviteCode:name:avatarUrl:completion:)` to identify the customer with the Streem `sharedInstance`. And finally you can `startRemoteStreem` with the remote user contained in the invitation. The whole flow looks like:
```

Comment:
See my comments on @callum-code's PR: https://github.com/streem/streem-app/pull/3740. I think we should simplify this flow and get rid of the separate `login()` call before we release the iOS SDK externally.

Reply:
I agree with @garyp, but maybe that's out of scope for this PR. Once @callum-code simplifies the flow, then we can adjust these instructions accordingly.

Reply:
I agree... a simple `startRemoteStreem(withInvitationCode:)` would be best.

Reply:
I agree that would be simpler. After talking with @callum-code we believe there's enough work involved to make that change that changing here is out of scope for this PR. Once we've landed on how `startRemoteStreem(withInvitationCode:)` works in its entirety we'll update the documentation here.

Reply:
[agreed](https://github.com/streem/streem-app/pull/3740#discussion_r427449194)

Reply:
1. That's fine to do this in a future PR. But I do think we shouldn't release the next SDK version until that work is done.

2. I'm not sure about collapsing it to a single `startRemoteStreem(withInvitationCode:)` call. I think there's value in keeping the different paths through the SDK similar to each other: you always `init`, `identify`, and then do whatever else you need to do. I could also see a use case for an app that wants to provide additional user profile info for the customer, which would require use of `identify`. So I'd lean towards having `identify(withInvitationCode:)` and then something like `startRemoteStreem(fromInvitation:)`.